mwrefs/utilities/fetch_references.py

```diff
@@ -3,13 +3,14 @@
 
 Usage:
     fetch_references -h | --help
-    fetch_references <host> <rev-id>
+    fetch_references <host> <title>
```

Comment:
rev_id is more useful than text because we often want to get references historically.